### ISSUE TITLE
Gonosomal inheritance

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7681]
**HPO term**
[Gonosomal inheritance](https://hpo.jax.org/app/browse/term/HP:0010985)

**Reason for deprecation**

Discussions with genCC team led to the conclusion that this label is not used in the community and that X and Y could just be children of the upper level inheritance term.

**Suggested replacement or term to consider**

**Comment by pnrobinson** [comment]
done
